src/main/groovy/net/wasdev/wlp/gradle/plugins/utils/ServerConfigDocument.java

```diff
@@ -236,44 +268,50 @@ private void parseConfigDropinsDir() throws Exception {
             File overrides = new File(configDropins, "overrides");
 
             if (overrides.exists()) {
-                File[] cfgFiles = overrides.listFiles();
-
-                for (int i = 0; i < cfgFiles.length; i++) {
-                    if (cfgFiles[i].isFile()) {
-                        parseDropinsFiles(cfgFiles[i]);
-                    }
-                }
+                parseDropinsFiles(overrides.listFiles());
             }
 
             File defaults = new File(configDropins, "defaults");
             if (defaults.exists()) {
-                File[] cfgFiles = defaults.listFiles();
+                parseDropinsFiles(defaults.listFiles());
+            }
+        }
+    }
 
-                for (int i = 0; i < cfgFiles.length; i++) {
-                    if (cfgFiles[i].isFile()) {
-                        parseDropinsFiles(cfgFiles[i]);
-                    }
-                }
+    private void parseDropinsFiles(File[] files) throws XPathExpressionException, IOException, SAXException {
+        for (int i = 0; i < files.length; i++) {
+            if (files[i].isFile()) {
+                parseDropinsFile(files[i]);
             }
         }
     }
 
-    private void parseDropinsFiles(File file) throws Exception {
-        // get input XML Document
+    private Document parseDropinsXMLFile(File file) throws FileNotFoundException, IOException {
+        try {
 
-        Document doc = parseDocument(new FileInputStream(file));
-        if(doc != null) {
-            //We are not calling parseApplication for configDropins server.xml documents configured by the plugin
-            //because we do not want to add locations defined here to the list of locations defined in the other server.xml documents.
-            if (!file.getName().equals(ApplicationXmlDocument.APP_XML_FILENAME)) {
-                parseApplication(doc, "/server/application | /server/webApplication | /server/enterpriseApplication");
+            FileInputStream is = new FileInputStream(file);
+            Document d = parseDocument(is);
+            is.close();
+            return d;
+            } catch (SAXException ex) {
+                //If the file was not valid XML, assume it was some other non XML file in dropins.
+                System.out.println("Dropins file " + file.getAbsolutePath() + " was not parseable as XML");
```

Comment:
Let's make this a log.debug() message.  We generally don't want this message to show up.

Can we change this message to something like:
"Dropins file " + file.getAbsolutePath() + " is not XML and is not evaluated for application configuration."